README.md

```diff
@@ -1,119 +1,231 @@
 Cartography
 ===========
 
+[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-url]][daviddm-image]
 [![Build Status](https://secure.travis-ci.org/Adslot/cartography.png?branch=master)](http://travis-ci.org/Adslot/cartography)
 
 Javascript Object to Object mapper
 
 Cartography takes an input Object and translates it into a new Object with a
-different structure according to a schema.
+different structure as specified by a schema.
 It can also be used for validating input structures.
 
-```coffeescript
-{same, from, map, filters, CartographyError} = require 'cartography'
-{required, isInteger, isOneOf} = filter
-
-isPositive = (v) -> if v > 0 then v else throw new CartographyError 'must be positive'
-
-cartographyUserSchema =
-  id: same isInteger
-  username: from 'email', required, (s) -> s.toLowerCase()
-  password: same required, isString
-  provider: -> 'Adslot'
-  maxShare: from 'sales.salePercentage', parseInt, isInteger, isPositive
-  address:
-    streetName: from 'addressStreet', required, isString
-    streetNumber: from 'addressNumber', required, isString
-    state: from 'addressState', isOneOf ['QLD', 'SA', 'ACT', 'NSW', 'VIC']
-
-userInDBFormat = map userInTransferFormat, cartographyUserSchema
+
+```javascript
+
+  var theInput = {
+    id: 123,
+    userName: 'HappyLand',
+    color: {
+      definition: '#7ff'
+    },
+    day: 'Mon',
+    month: 'Aug',
+  }
+
+  var theDesiredOutput = {
+    id: 123,
+    name: 'HappyLand',
+    base: {
+      color: '#7FF',
+      time: 'Mon - Aug'
+    }
+  }
+
+  // enters Cartography
```

Comment:
I guess it was theatrical.
Would `// let's use Cartography` be more clear?
Shall I remove the comment entirely?